index.css

```diff
@@ -2,4 +2,135 @@
 @import url(login/login.css);
 @import url(productPage/productLanding.css);
 @import url(signUp/sign-up.css);
-@import url(wishlist/wishlist.css);
\ No newline at end of file
+@import url(wishlist/wishlist.css);
+
+html .without-shadow {
+    box-shadow: none;
+}
+
+.nav-bar {
+    position: sticky;
+    top: 0;
+    z-index: 4;
```

Comment:
I think it was mentioned in one of the code reviews, that if you are keeping navbar sticky, give it max z-index.
You can consider doing that, give it a CSS variable instead of direct value

Reply:
As Vikram sir mentioned in his thread that you should be aware of the maximum z-index you are using and should not put any arbitrary value just to push it on top of everything.
Therefore on the same grounds, I am aware that I have not used z-index>2 anywhere else. Therefore I have left just 1 z-index stack for sudden use case and have assigned a proper value to it.